tests/tests_pytorch/accelerators/test_tpu.py

```diff
@@ -66,14 +69,16 @@ def test_resume_training_on_cpu(tmpdir):
 
 
 @RunIf(tpu=True)
+@mock.patch.dict(os.environ, {}, clear=True)
 def test_if_test_works_after_train(tmpdir):
     """Ensure that .test() works after .fit()"""
-
-    # Train a model on TPU
     model = BoringModel()
     trainer = Trainer(max_epochs=1, accelerator="tpu", devices=8, default_root_dir=tmpdir, fast_dev_run=True)
-    trainer.fit(model)
-    assert len(trainer.test(model)) == 1
+    # https://github.com/pytorch/xla/issues/1666
+    with pytest.raises(ProcessRaisedException, match="OSError: libmkl_intel_lp64"):
```

Comment:
I'm hoping that unpinning `mkl` will resolve this after the new docker image is published